
[34m[1mLogs will be synced with wandb.
Traceback (most recent call last):
  File "train.py", line 110, in <module>
    train(parse_cfg(Path().cwd() / __CONFIG__))
  File "train.py", line 71, in train
    buffer += episode
  File "/home/jeongtae/RLMPPI/src/algorithm/helper.py", line 242, in __add__
    self.add(episode)
  File "/home/jeongtae/RLMPPI/src/algorithm/helper.py", line 260, in add
    self._reward[self.idx:self.idx + episode_length] = episode.reward()
TypeError: 'Tensor' object is not callable